ngraph/frontend/tensorflow/src/op/elu.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <default_opset.h>
+
+#include <op_table.hpp>
+#include <tensorflow_frontend/node_context.hpp>
```

Comment:
"node_context" because this is internal header

Reply:
fixed